.github/workflows/close-inactive-issues.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
       - uses: actions/stale@v7
         with:
           debug-only: true # TODO: Delete this line when we are ready to implement this for our process.
+          exempt-issue-assignees: "jacob314,kenzieschmoll,CoderDake,polina-c"
```

Comment:
@jacob314 sorry about my confusion on this. I assumed that since there is a github provided action, that using it would be preferred.
For intellij, they [this 3rd party action](https://github.com/lee-dohm/no-response).
Are you ok with us using that same one?